sdk/ai/azure-ai-resources/azure/ai/resources/entities/ai_hub.py

```diff
@@ -382,18 +382,18 @@ def primary_user_assigned_identity(self, value: str):
     # Read only so no setter.
     @property
     def enable_data_isolation(self) -> bool:
-        """Whether or not data isolation is enabled for the workspace hub.
+        """Whether or not data isolation is enabled for the AI hub.
 
-        :return: Boolean value indicating whether or not data isolation is enabled for the workspace hub.
+        :return: Boolean value indicating whether or not data isolation is enabled for the AI hub.
         :rtype: bool
         """
         return self._workspace_hub.enable_data_isolation
 
     @property
     def default_project_resource_group(self) -> str:
-        """The default project's resource group.
+        """The default project's AI group.
```

Comment:
@MilesHolland - This looks like a find/replace error. Should remain resource group.